src/main/java/org/osm2world/core/util/ConfigUtil.java

```diff
@@ -130,4 +133,24 @@ public static void parseFonts(Configuration config) {
 		}
 	}
 
+	/**
+	 * If config references some files by path i.e. textures
+	 * resolve file paths relative to config location
+	 */
+	public static File resolveFileConfigProperty(Configuration config, String fileName) {
+		if (fileName == null) {
+			return null;
+		}
+
+		String basePath = config.getString("configPath", null);
+		File file = new File(basePath, fileName);
```

Comment:
This does not yield correct results for absolute paths in fileName. For example, I tested it unsuccessfully with `srtmDir = /data/srtm`.